servicex/uproot_raw/uproot_raw.py

```diff
@@ -62,3 +64,13 @@ def generate_selection_string(self):
         else:
             final_query = self.query
         return json.dumps([json.loads(_.model_dump_json()) for _ in final_query])
+
+    @classmethod
```

Comment:
What does this do? In particular it's not mandatory that every query is a TreeSubQuery?

Reply:
This allows YAML interface like other queries with yaml tag. You're correct as it assumes it's a `TreeSubQuery`. I added minimal things for now.

Reply:
Add support for `CopyHistogramSubQuery` by [3c7f25c](https://github.com/ssl-hep/ServiceX_frontend/pull/401/commits/3c7f25c0cb6bf8638127a40020db444b3b81e8d5)